src/views/welcomeView.js

```diff
@@ -7,10 +7,12 @@ import { START_QUIZ_BUTTON_ID } from '../constants.js';
 export const createWelcomeElement = () => {
   const element = document.createElement('div');
   element.innerHTML = String.raw`
-    <h1>Welcome</h1>
-    <label for="name" class="name-label">Enter your name:</label>
-    <input type="text" id="name" name="name" required class="name-input">
-    <button id="${START_QUIZ_BUTTON_ID}" class="button-style">start quiz</button>
+  <h1 class="welcome-page-heading">WELCOME</h1>
+  <label for="name" class="name-label">What should we call you?</label>
+  <input type="text" id="name" name="name" required class="name-input">
+  <button id="${START_QUIZ_BUTTON_ID}" class="button-style">START</button>
+  <img class="toad" src="https://upload.wikimedia.org/wikipedia/en/d/d1/Toad_3D_Land.png" >
+  <img class="toad1" src="https://upload.wikimedia.org/wikipedia/en/d/d1/Toad_3D_Land.png" >
```

Comment:
With these imagers, it would be better to download them and reference them from the assets folder. This is for a couple reasons: by referencing outside websites, it can be prone to 'link rot' where links outside of your own servers could go down. It is also more efficient to reference from a file stored within the project.